MoltenVK/MoltenVK/GPUObjects/MVKPipeline.h

```diff
@@ -218,6 +218,49 @@ struct MVKStagedDescriptorBindingUse {
 	MVKBitArray stages[4] = {};
 };
 
+/** Enumeration identifying different state content types. */
+enum MVKRenderStateType {
```

Comment:
Should this be an `enum class`? I guess it doesn't matter so much since this isn't a public API...

Reply:
Yeah. Our pattern for internal `enums` so far is to just use `enum`.